sdcm/cluster.py

```diff
@@ -1596,6 +1599,27 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         if not res or "Active: failed" in res.stdout:
             raise ScyllaManagerError("Scylla-Manager is not properly installed or not running: {}".format(res))
 
+        self.start_scylla_manager_log_capture()
+
+    def retrieve_scylla_manager_log(self):
+        mgmt_log_name = os.path.join(self.logdir, 'scylla_manager.log')
+        cmd = "sudo journalctl -u scylla-manager -f".format(mgmt_log_name)
+        self.remoter.run(cmd, ignore_status=True, verbose=True, log_file=mgmt_log_name)
+
+    def scylla_manager_log_thread(self):
+        while not self.termination_event.isSet():
+            self.retrieve_scylla_manager_log()
+
+    def start_scylla_manager_log_capture(self):
+        self._scylla_manager_journal_thread = threading.Thread(target=self.scylla_manager_log_thread)
+        self._scylla_manager_journal_thread.start()
+
+    def stop_scylla_manager_log_capture(self, timeout=10):
+        cmd = "sudo pkill -f \"sudo journalctl -u scylla-manager -f\""
```

Comment:
better to avoid escaping where is possible: 
`'sudo pkill -f "sudo journalctl -u scylla-manager -f" '`